ext/phar/phar_object.stub.php

```diff
@@ -13,52 +13,52 @@ public function __construct(string $filename, int $flags = FilesystemIterator::S
     public function __destruct() {}
 
     /** @return void */
-    public function addEmptyDir(string $dirname) {}
+    public function addEmptyDir(string $directory) {}
 
     /** @return void */
-    public function addFile(string $filename, ?string $localname = null) {}
+    public function addFile(string $filename, ?string $localName = null) {}
 
     /** @return void */
-    public function addFromString(string $localname, string $contents) {}
+    public function addFromString(string $localName, string $contents) {}
 
     /** @return array|false */
-    public function buildFromDirectory(string $base_dir, string $regex = "") {}
+    public function buildFromDirectory(string $directory, string $pattern = "") {}
 
     /** @return array|false */
-    public function buildFromIterator(Traversable $iterator, ?string $base_directory = null) {}
+    public function buildFromIterator(Traversable $iterator, ?string $stripDirectory = null) {}
```

Comment:
Hmm, `$stripDirectory` is a bit strange for me. I think I'd be fine with `$baseDirectory` or `$basePath` .

Reply:
Okay, I reverted this to the `$baseDirectory` name.